package.json

```diff
@@ -116,7 +125,6 @@
     "c8": "^7.11.2",
     "conventional-changelog-cli": "^2.1.1",
     "d3-drag": "^2.0.0",
```

Comment:
d3-drag is included in the normal dependency, so remove from the dev dependencies?